dist/handsontable.full.js

```diff
@@ -90184,8 +90184,14 @@ var _default = function _default() {
     maxCols: Infinity,
```

Comment:
I'm not sure if I mentioned that before (sorry if not🙈).
We do not push modified files in `dist/` on feature branches (you can read more about contributing in https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md).

Would you mind reverting changes in this file?

Reply:
@swistach - my bad, I read the contributing guidelines and build docs but forgot ;) Reverted now.